nise/generators/gcp/project_generator.py

```diff
@@ -1,30 +1,44 @@
 """Module for generating GCP Projects."""
+from random import choice
+
 from faker import Faker
 
 
 class ProjectGenerator:
     """Generator for GCP Compute Engine data."""
 
+    PROJECT_INFO = (  # id -  name, labels, ancestry_numbers
+        ("doug-cost-test", "Doug-cost-test", "[]", ""),
```

Comment:
I think we may want to rename the default project names here. We may be able to do something auto generated. 🤷‍♂️

Reply:
addressed here: https://github.com/project-koku/nise/pull/314/commits/3e23ad4e34e94d146ab961688c6555c6203673ff